### PR TITLE
feat: upgrade to webpack 5 and node 16

### LINKED ISSUE #20: Add support for node 16 to build
- [ ] Update the build workflow to use a matrix and test on node versions: 12, 14, and 16
- [ ] Update the docker container to default to node 16
- [ ] test running the node-16 docker container with the orientdb and graphkb client containers

### PR DESCRIPTION
**Pull request by areisle**
closes #20
this required no longer supporting scss modules (only the theme file used them)
and swapping from jsonwebtoken to a browser friendly decoding option (jwt-decode) since webpack 5
no longer automatically polyfills standard node libraries and
jsonwebtoken relies on many of them

### CODE REVIEW

File: config/webpack/common.js [hunk 1]
```diff
@@ -221,26 +192,24 @@ const createBaseConfig = ({
...
+          errors: true,
+          warnings: false,
+        },
+      },
     },
     performance: { hints: 'warning' },
-    // production optimizations
     optimization: {
       runtimeChunk: 'single',
-      minimizer: [
```
Comment (creisle) [inline config/webpack/common.js:234]: Do we not need a minimizer anymore?

**Comment by creisle** [comment]
I went through all views and everything I tested looked the same as current dev, no obvious differences

**Review by creisle — APPROVED** [review]

File: config/webpack/common.js (hunk 1 above)
Comment (areisle) [reply config/webpack/common.js:234]: webpack 5 comes with [terser builtin without any configuration necessary](https://webpack.js.org/plugins/terser-webpack-plugin/#:~:text=Webpack%20v5%20comes%20with%20the%20latest%20terser%2Dwebpack%2Dplugin%20out%20of%20the%20box.)

**Comment by creisle** [comment]
Note: having some issues updating the bamboo build to node 16 internally https://www.bcgsc.ca/bamboo/browse/PORI-GKBC32-6

**Pull request merged**